packages/wrangler/package.json

```diff
@@ -60,7 +60,7 @@
 		"emit-types": "tsc -p tsconfig.emit.json && node -r esbuild-register scripts/emit-types.ts",
 		"prepublishOnly": "SOURCEMAPS=false npm run build",
 		"start": "pnpm run bundle && cross-env NODE_OPTIONS=--enable-source-maps ./bin/wrangler.js",
-		"test": "pnpm run assert-git-version && jest",
+		"test": "pnpm run assert-git-version && jest --runInBand",
```

Comment:
Had to do this to avoid a race condition where every test thought it could start the `InspectorProxyWorker` on `9229` because `get-port` doesn't work across processes (jest processes this case)

We confirmed this affects `wrangler dev` and `unstable_dev` if you run multiple processes in parallel where `get-port` thinks the requested port is available but then there's a race for all the workerd instances to bind to it

We have a fix coming in a follow-up

Reply:
TODO: make miniflare surface "port in use" errors so wrangler can retry instantiating Miniflare with port 0